Parsers/Binary To Decimal Converter.js

```diff
@@ -0,0 +1,47 @@
+/*
+activation_example:!converttoDecimal <binary_code>
+regex:^!converttoDecimal 
```

Comment:
Please remove this capitalisation, as this adds confusion when compared to all other parsers. Additionally, it is not clear that this parser converts from binary to decimal. It might be worthwhile making it something like `!bin2dec`

Reply:
ok I wil make changes